packages/soundxyz/src/Soundxyz.test.ts

```diff
@@ -161,3 +161,23 @@ describe('getProjectFees', () => {
     expect(fee).toEqual(BigInt('777000000000000'))
   })
 })
+describe('getFees', () => {
```

Comment:
These tests arn't super meaningful but this is in line with the standard of tests for this function so NBD, I can clean these up at the same time as those.